spec/std/llvm/llvm_spec.cr

```diff
@@ -21,9 +21,7 @@ describe LLVM do
 
   it ".default_target_triple" do
     triple = LLVM.default_target_triple
-    {% if flag?(:darwin) %}
-      triple.should match(/-apple-macosx$/)
```

Comment:
**question:** Could we adjust the expectation accordingly?
```suggestion
      triple.should match(/-apple-(darwin|macosx)/)
```

Reply:
Yeah, fair enough. Fixed in https://github.com/crystal-lang/crystal/pull/14466/commits/d4db294766b1db8b4675be25a4edf86e70d01b35. 🙂